src/twisted/conch/ssh/transport.py

```diff
@@ -24,6 +24,15 @@
 from cryptography.hazmat.primitives import serialization
 from cryptography.hazmat.primitives.asymmetric import dh, ec, x25519
 from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
+
+try:
+    from cryptography.hazmat.decrepit.ciphers import (  # type: ignore[attr-defined]
+        TripleDES,
+    )
+except ImportError:
+    # Deprecated path, will be removed in cryptography 48.0.0
+    from cryptography.hazmat.decrepit.ciphers.algorithms import TripleDES
```

Comment:
This is the new path, which should be in line 29. The deprecated path is `from cryptography.hazmat.primitives.ciphers.algorithms`